### PR TITLE
Added Student Image

### PR DESCRIPTION
**Pull request by dewanshrawat15**
I've added my image in the students section.

### CODE REVIEW

File: CNAME [hunk 1]
```diff
@@ -1 +1 @@
-gci16.fossasia.org
+gci16.fossasia.org/
```
Comment (jigyasa-grover) [inline CNAME]: You arent supposed to change this file.

File: _data/students.yml [hunk 2]
```diff
@@ -1,4 +1,4 @@
-﻿# New members please add your credentials to the bottom of the file.
+- # New members please add your credentials to the bottom of the file.
```
Comment (jigyasa-grover) [inline _data/students.yml]: This change wasnt required too..

File: _data/students.yml [hunk 3]
```diff
@@ -326,3 +326,10 @@
   twitter: singhprabh2929
   facebook: Prabhdeep Singh
   bio: 14, Programmer in every way
+  
+ - name: Dewansh Rawat
+  github: dewanshrawat15
```
Comment (jigyasa-grover) [inline _data/students.yml:347]: Improper indentation, refer previous entries.

File: index.html [hunk 4]
```diff
@@ -1,5 +1,6 @@
 ---
 ---
+<!-- I am cool-->
```
Comment (jigyasa-grover) [inline index.html]: Why was this added ? 😅

**Review by jigyasa-grover — CHANGES_REQUESTED** [review]

**Comment by jigyasa-grover** [comment]
* Squash all your commits into one and rename it to something meaningful, such as "Adding <-your-name-> in students/mentors section"
* Also Provide the link to live `gh-pages` from your forked repository or relevant screenshot for easier review.
* Resolve merge conflicts, this [How To Manually Fix Git Merge Conflicts (video)](https://www.youtube.com/watch?v=g8BRcB9NLp4) might be helpful. 

Cheers !

**Review by abishekvashok — CHANGES_REQUESTED** [review]
Please fix as stated by @jig08 and remove the unwanted lines, bro we know you are cool so no need to imply it 👍 😂

**Comment by dewanshrawat15** [comment]
Sorry about that.😂 I'm currently working on that fix only...
What I'm unable to understand is how do I fix the image..?? This type of
HTML structure is a first for me and I'm unable to put my image up. Could
you possibly help me..??

**Comment by ankitrgadiya** [comment]
@dewanshrawat15 You don't have to touch index.html in order to put image. Go ahead and add your image name in students.yml as specified [here](https://github.com/fossasia/gci16.fossasia.org/#adding-yourselves-to-student-section).

File: index.html (hunk 4 above)
Comment (m1guelpf) [reply index.html]: I think you should remove this :sweat_smile:

**Comment by dewanshrawat15** [comment]
****
***

**Comment by dewanshrawat15** [comment]
I have done the required changes.

**Comment by abishekvashok** [comment]
Great work, now can you fix the merge conflicts by following the tutorials in the contributing.md file

**Comment by animeshsinghweb** [comment]
@dewanshrawat15 Also there are some new conflicts problem in your `students.yml` file.
That's the reason it is suggest to isolate from the main branch (in this case it is **gh-pages** branch, make changes in your new branch, commit, recheck, squash, and then PR).
I'd even suggest you, from next time while requesting for Pull Request make a new branch using `git checkout -b yourNewBranchName` and then you can do your changes there, commit, squash and request for PR.

**Comment by dewanshrawat15** [comment]
Um, I guess I've done the required changes on my repository. What now..??

**Comment by animeshsinghweb** [comment]
@dewanshrawat15.
There is yet some conflict problem.
If you need Personal Assistant please DM me at Slack id animeshsinghweb

**Comment by m1guelpf** [comment]
@dewanshrawat15 LGTM

**Comment by dewanshrawat15** [comment]
Good Evening
I just received a notification mail from github about the merge. Does that
mean my image will also appear on gci16.fossasia.org starting now..??

Regards
Dewansh

**Comment by m1guelpf** [comment]
@dewanshrawat15 Yep

**Comment by dewanshrawat15** [comment]
Don't know why I don't see it..😂

**Comment by m1guelpf** [comment]
@dewanshrawat15 Build was broken, fixed now

**Pull request merged**